lib/array_memory.coffee

```diff
@@ -25,7 +25,7 @@ class ArrayMemory
     @brain.get('lunchnlearn') || []
 
   any: ->
-    _.any(@ideas())
+    _.some(@ideas())
 
   toString: (joinWith = EOL)->
```

Comment:
wow, coffee script. 


![What year is it?!](https://media0.giphy.com/media/116Ml7ECYdtZ8Q/giphy.gif)

Reply:
1999